test/blackbox-tests/test-cases/variants/variants-using/dune-project

```diff
@@ -0,0 +1,3 @@
+(lang dune 1.1)
+
+(using variants 1.0)
```

Comment:
Maybe a simple way to declare intent that this is not completely stable is to use a `0.x` version here.